test/ui/LoginPage.ts

```diff
@@ -0,0 +1,41 @@
+import { By, WebDriver, WebElement } from 'selenium-webdriver';
+import BasePage from './BasePage';
+
+
+export class LoginPage extends BasePage{
+  
+  private url: string = "https://5chmbvngab.eu-west-1.awsapprunner.com/loginform";
```

Comment:
When we can get secrets added into the github actions this can be updated to something similar to this:

  `
    private UI_TEST_URL = process.env.UI_TEST_URL;
    private HOSTED_URL = process.env.API_URL;
    private url: string;

    constructor() {
      super();
      this.url = `${this.HOSTED_URL}/job-roles` || `${this.UI_TEST_URL}/job-roles`;
      console.log("UI_TEST_URL:", this.UI_TEST_URL);
      console.log("HOSTED_URL:", this.HOSTED_URL);
      console.log("Constructed URL:", this.url);
    }
`

Currently unable to obtain these.

Reply:
Just coming back to this, I think implementing something like this is reasonable:
private baseUrl: string = process.env.UI_TEST_URL || 'https://5chmbvngab.eu-west-1.awsapprunner.com';
  private urlPath: string = '/loginForm';
  private url: string = this.baseUrl+this.urlPath;

This will check that if the UI_TEST_URL is defined, and if not, it will run on the AWS server.